events/class_tests/meeting_tests.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+from django.utils import timezone
+from .event_tests import new_event  # noqa: F811 ,F401
+from django.core.exceptions import ValidationError
+from ..models import PossibleMeeting, EventParticipant
+from .participant_test import event_participant_not_creator  # noqa:F811, F401
+from users.tests import user0  # noqa: F401
+
+import pytest
+
+DATE_TIME_START = datetime(2022, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
+DATE_TIME_END = datetime(2022, 3, 24, 14, 12, 12, 0, tzinfo=timezone.utc)
+
+
+@pytest.fixture
+def possible_meeting0(event_participant_not_creator):  # noqa: F811
+    return PossibleMeeting(participant_id=event_participant_not_creator,
+                           date_time_start=DATE_TIME_START, date_time_end=DATE_TIME_END)
+
+
+@pytest.fixture
+def persist_possible_meeting(possible_meeting0):
+    possible_meeting0.participant_id.user_id.save()
+    possible_meeting0.participant_id.event_id.save()
+    possible_meeting0.participant_id.save()
+    possible_meeting0.save()
+    return possible_meeting0
+
+
+@pytest.mark.django_db
+def test_persist_possible_meeting(persist_possible_meeting):
+    assert persist_possible_meeting in PossibleMeeting.objects.all()
+
+
+@pytest.mark.django_db
+def test_delete_persist_possible_meeting(persist_possible_meeting):
+    persist_possible_meeting.delete()
+    assert persist_possible_meeting not in PossibleMeeting.objects.all()
+
+
+@pytest.mark.django_db
+def test_delete_participant_deletes_possible_meeting(persist_possible_meeting):
+    persist_possible_meeting.participant_id.delete()
+    assert persist_possible_meeting not in PossibleMeeting.objects.all()
```

Comment:
Because of the wide use in @pytest.mark.django_db
I would suggest creating a class for all the tests in this file and placing this decorator above the class.
```suggestion
@pytest.mark.django_db
class MeetingTests():
   pass 
```

Reply:
Done